internal/bminventory/inventory_v2_handlers.go

```diff
@@ -638,7 +693,14 @@ func (b *bareMetalInventory) infraEnvIPXEScript(ctx context.Context, infraEnv *c
 		return "", errors.Wrap(err, "failed to sign initrd URL")
 	}
 
-	return fmt.Sprintf(ipxeScriptFormat, initrdURL, kernelURL, rootfsURL), nil
+	return fmt.Sprintf(ipxeSecondScriptFormat, initrdURL, kernelURL, rootfsURL), nil
+}
+
+func (b *bareMetalInventory) infraEnvIPXEScript(ctx context.Context, infraEnv *common.InfraEnv, mac *strfmt.MAC, urlPath string) (string, error) {
+	if mac == nil || len(*mac) == 0 {
+		return b.firstIPXEScript(urlPath)
+	}
```

Comment:
This means that we *have* to pass a host mac to get the script at all.
I thought we talked about retaining a way to query for the script without requiring a host.

Reply:
We can perform the query with dummy mac.  It will provide the script.

Reply:
OK. Changed